test/diff_type_struct_ptr/test__.c

```diff
@@ -0,0 +1,28 @@
+#include <stdio.h>
```

Comment:
Very weird filename! In general, it would be nice to change the names of these tests so that they briefly indicate what they are testing (i.e., they aren't just "test1.c", "test2.c", etc., but instead the names have a tiny bit of semantics). As it is, it'll be hard to find the tests for particular features later (although the test directories definitely help!).

Reply:
Ah, you were not supposed to see these files! This "test" was me playing with struct pointers of different types, which made me conclude that I won't be aliasing them. I remembered to not push these changes at first, but I forgot to do that this time. I'll put tests that are just for me in a different folder and is .gitignore'd